StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+PaymentMethodAvailability.swift

```diff
@@ -172,11 +175,36 @@ extension PaymentSheet {
                 return "financialConnectionsSDK: The FinancialConnections SDK must be linked. See https://stripe.com/docs/payments/accept-a-payment?platform=ios&ui=payment-sheet#ios-ach"
             case .validUSBankVerificationMethod:
                 return "Requires a valid US bank verification method."
+            case .instantBankPaymentRequirement(let requirement):
+                return requirement.debugDescription
             }
         }
 
     }
 
+    enum InstantBankPaymentRequirement: Hashable {
+        case missingLink
+        case unexpectedUsBankAccount
+        case invalidEmailCollectionConfiguration
+        case linkFundingSourcesMissingBankAccount
+        case unexpectedLinkMode
+
+        var debugDescription: String {
+            switch self {
+            case .missingLink:
+                return "Specified payment methods must contain link"
+            case .unexpectedUsBankAccount:
+                return "US Bank Account should not be set as a payment method"
+            case .invalidEmailCollectionConfiguration:
+                return "The provided configuration must either collect an email, or a default non-empty email must be provided"
+            case .linkFundingSourcesMissingBankAccount:
+                return "Link funding sources must contain bank account"
+            case .unexpectedLinkMode:
+                return "The Link Mode received is not Link Card Brand"
+            }
+        }
```

Comment:
We should reword these a little. And with the other suggestions, we could probably drop two cases that would be covered by returning `.notSupported`.

```suggestion
        var debugDescription: String {
            switch self {
            case .unexpectedUsBankAccount:
                return "The list of payment method types includes 'us_bank_account', which prevents the 'Bank' tab from being displayed."
            case .invalidEmailCollectionConfiguration:
                return "The provided configuration must either collect an email, or a default non-empty email must be provided."
            case .linkFundingSourcesMissingBankAccount:
                return "Your account isn't set up to process Instant Bank Payments. Reach out to Stripe support."
            }
        }
```

Reply:
Much better than what I had - will update!